Makefile

```diff
@@ -1,36 +1,26 @@
 .PHONY: all binary ut clean
 
-SRCDIR=calico_kubernetes
-BUILD_DIR=build_calico_kubernetes
-BUILD_FILES=$(BUILD_DIR)/Dockerfile $(BUILD_DIR)/requirements.txt
-
+SRCFILES=$(shell find calico_kubernetes)
 
 default: all
 all: binary test
 test: ut
+binary: dist/calico_kubernetes
 
-# Build a new docker image to be used by binary or tests
-kubernetesbuild.created: $(BUILD_FILES)
-	cd build_calico_kubernetes; docker build -t calico/kubernetes-build .
-	touch kubernetesbuild.created
-
-binary: kubernetesbuild.created
+dist/calico_kubernetes: $(SRCFILES)
 	mkdir -p dist
 	chmod 777 `pwd`/dist
 	
 	# Build the kubernetes plugin
 	docker run \
 	-u user \
-	-v `pwd`/calico_kubernetes:/code/calico_kubernetes \
-	-v `pwd`/dist:/code/dist \
-	-e PYTHONPATH=/code/calico_kubernetes \
-	calico/kubernetes-build pyinstaller calico_kubernetes/calico_kubernetes.py -a -F -s --clean
+	-v `pwd`:/code \
+	calico/build pyinstaller calico_kubernetes/calico_kubernetes.py -a -F -s --clean
```

Comment:
What is the recommended way to add new requirements if we're using a centralized build container?